### ISSUE TITLE
Revive dashboard overlay doesn’t work...

### ISSUE THREAD

**Issue by mapotter99** [LibreVR/Revive#816]
I haven’t had time to use VR much recently. I went to use it today, and the Revive dashboard won’t open. The button is present, but when I click it, the dashboard never opens. 

This same problem is also affecting OpenVR Advanced Settings... button present, but it doesn’t open when clicked..., which leads me to believe this is a problem with SteamVR.

I have tried both the release and beta versions of both SteamVR and Revive (1.1.14 and 1.2.0 Beta 2).

**Comment by mapotter99** [comment]
A clarification...

The UI seems to be invisible. Clicking where the UI should be opens an Oculus application. 

Video card driver is the latest nVidia GeForce release 388.31.

**Comment by Armada651** [comment]
What OS are you using?

**Comment by mapotter99** [comment]
Fully updated Windows 10 Pro with Fall Creators Update.

**Comment by Armada651** [comment]
1. Press WinKey + R
2. Enter `%APPDATA%\Revive`, press OK.
3. Drag `ReviveOverlay.txt` into the comment box here and post the comment.

**Comment by mapotter99** [comment]
[ReviveOverlay.txt](https://github.com/LibreVR/Revive/files/1484621/ReviveOverlay.txt)

**Comment by Armada651** [comment]
Doesn't seem to be any clear error, I'll get back to this issue after the next release.

**Comment by mapotter99** [comment]
This issue seems to be the same as issue #709. 

Not sure why it just started showing up for me. 

I have tried assigning Revive to use the GTX 1070 instead of “automatic”. This didn’t resolve the problem.

**Comment by Armada651** [comment]
> I have tried assigning Revive to use the GTX 1070 instead of “automatic”.

Wait, are you using a multi-GPU setup?

**Comment by mapotter99** [comment]
The Intel chipset has Intel HD 530 integrated graphics built in.

I’m using an Alienware X51 R3.

**Comment by Armada651** [comment]
Hmm, it would make sense if the Qt application is rendering on the Intel GPU and SteamVR is on the dedicated GPU. In that case Revive can't send the texture to SteamVR.

**Comment by mapotter99** [comment]
I've determined the source of this problem for my occurrence... Windows 10's HDR & Advanced Color setting.  I turned this off, and the UI for both Revive and Advanced Settings displayed properly.

I recently bought a new 4K HDR TV for my basement, which also serves as the monitor for my gaming PC.  I just happened to think to try disabling HDR & Advanced Color in the Windows 10 Display options.

HOORAY!
